edc-client-dotnet-test/EdcClientSingletonTest.cs

```diff
@@ -0,0 +1,102 @@
+using edc_client_dotnet;
+using edc_client_dotnet.model;
+using Newtonsoft.Json;
+
+namespace edc_client_dotnet_test;
+
+[TestClass]
+public class EdcClientSingletonTest
+{
+    private static string DEFAULT_URL = "https://demo.easydoccontents.com/help/home";
+
+    [TestInitialize]
+    public void Setup()
+    {
+        EdcClientSingleton.GetInstance().SetServerUrl("https://demo.easydoccontents.com");
```

Comment:
Why did you not use the Constant?
Either you define a constant and you use it or you write independant unit tests.